core/src/main/scala/org/bykn/bosatsu/library/LibConfig.scala

```diff
@@ -0,0 +1,62 @@
+package org.bykn.bosatsu.library
+
+import org.bykn.bosatsu.{Json, PackageName}
+import _root_.bosatsu.{TypedAst => proto}
+
+case class LibConfig(
+  name: String,
+  repoUri: String,
+  exportedPackages: List[PackageName],
+  internalPackages: List[PackageName],
```

Comment:
Instead of only internal this should be the set of everything and allow patterns so you don't have to edit this file every time you make a new package.